OpenRA.Game/Widgets/Widget.cs

```diff
@@ -347,7 +347,7 @@ void ForceYieldKeyboardFocus()
 				Ui.KeyboardFocusWidget = null;
 		}
 
-		public virtual string GetCursor(int2 pos) { return "default"; }
+		public virtual string GetCursor(int2 pos) { return ChromeMetrics.Get<string>("WidgetDefaultCursor"); }
```

Comment:
I wanted to you change this, specifically, to "DefaultCursor" because it *is* the default cursor. The DefaultCursor you currently define is afaik only used in exceptional cases, which may change as things are refactored in the future (e.g. the "real" load screen will be using DefaultCursor, but the "fake" load screen when loading a saved game will be using "WidgetDefaultCursor").

Unless there is a specific need to split these, it makes future maintenance simpler if we can keep a single "DefaultCursor" used in both places.

Do you have a specific need

Reply:
Where you want `WidgetDefaultCursor` used then?

Reply:
I can't quite follow, but I did the change.

Reply:
If you want you could test this by going back to the previous PR version and then setting different values for `DefaultCursor` versus `WidgetDefaultCursor`. You would find that `WidgetDefaultCursor` is used everywhere.